src/util/date.ts

```diff
@@ -21,7 +21,7 @@ const dateToStr = (date: Date): string => {
  * Validity duration is obtained from config.vaccination.validityInDays.
  * @example
  * calcExpiry(new Date("2021-03-31T16:01:00.000Z"));
- * // 1 Apr 2021 + 14 days = 15 Apr 2021 (SGT)
+ * // 1 Apr 2021 + 14 days = 15 April 2021 (SGT)
```

Comment:
What happens when the validityInDays config is not set? Also indicate in the ReadMe how to set this.

Reply:
`validityInDays` is currently fixed in this repo:
https://github.com/Notarise-gov-sg/sns-notify-recipients/blob/master/src/config.ts#L33

I couldn't find any similar config for it in the `api-vaccine` repo so I have it statically set it here first. I can look to upgrading this repo in the future to specify a config object (instead of env variables) so it's more explicit to the caller.

Reply:
Added a techdebt to track this. https://www.pivotaltracker.com/story/show/177941325